@ecomplus/storefront-components/src/js/TheProduct.js

```diff
@@ -505,6 +505,7 @@ export default {
   created () {
     if (this.product) {
       this.body = this.product
+      this.qntToBuy = this.product.min_quantity || this.qntToBuy
```

Comment:
Tá, mas aí c fez de um jeito preguiçoso né 🙃 
Não dá mais erro, mas também não funciona com hidratação, nem com se o objeto do produto não for enviado como prop.